salesagility/SuiteCRM
# Fix/duplicate bulk box

When selecting items for bulk action, the bulk action button was showing twice on both top and bottom. This fix stops this from happening.

## Description



The fix stops the bulk actions button from being shown twice.

## How To Test This

1) Navigate to a module with a table where a bulk action can be performed, e.g. the contacts module
2) Select one or more items in the list
3) Deselect the item/s
4) The 'Bulk Action' should only be shown once

## Types of changes


### Final checklist
